tests/tools/oesign/test-enclave/host/host.c

```diff
@@ -93,7 +94,7 @@ int main(int argc, const char* argv[])
         }
     }
 
-    if (_is_kss_supported())
+    if (_is_kss_supported() && oe_has_sgx_quote_provider())
```

Comment:
Suggest adding comments to explain the workaround. We can revisit this later as @CodeMonkeyLeet  suggested.

```suggestion
    /* check_kss_extended_ids currently assumes the quote provider is available.
     * Skip if there is no quote provider for now.
     */
    if (_is_kss_supported() && oe_has_sgx_quote_provider())
```